docs/marionette.region.md

```diff
@@ -261,6 +261,7 @@ In addition to adding and removing regions there are a few
 methods to help utilize regions.
 
 - `getRegion(name)` - Request a region from a view by name.
+ - Note: If the view hasn't been rendered at this point, it will be.
 - `getRegions()` - Returns an object literal of all regions on the view organized by name.
```

Comment:
It'll require a code change too.  `getRegions` isn't calling `render` or `getRegion`